notebooks/2022_04_05_PRL_explore_acs_data.ipynb

```diff
@@ -60,8 +60,8 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "\u001b[0m\u001b[01;31mcsv_hus.zip\u001b[0m  \u001b[01;32mpsam_husa.csv\u001b[0m*  \u001b[01;32mpsam_husc.csv\u001b[0m*\r\n",
-      "\u001b[01;31mcsv_pus.zip\u001b[0m  \u001b[01;32mpsam_husb.csv\u001b[0m*  \u001b[01;32mpsam_husd.csv\u001b[0m*\r\n"
+      "\u001B[0m\u001B[01;31mcsv_hus.zip\u001B[0m  \u001B[01;32mpsam_husa.csv\u001B[0m*  \u001B[01;32mpsam_husc.csv\u001B[0m*\r\n",
+      "\u001B[01;31mcsv_pus.zip\u001B[0m  \u001B[01;32mpsam_husb.csv\u001B[0m*  \u001B[01;32mpsam_husd.csv\u001B[0m*\r\n"
```

Comment:
This was a black formatting update from rebasing.

Reply:
Hmmm, weird. On a quick skim of [the Black code style](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html), I don't see reference to this behavior (it does talk about keeping the \u prefix lowercase), but I doubt it is problematic.

Reply:
I think this must be because you locally installed black [with the jupyter extra](https://black.readthedocs.io/en/stable/getting_started.html#installation). The rest of us/CI don't have this extra.